config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,20 @@
+# Settings for the collector audit payloads function.
```

Comment:
@brianhlin - we tend to have config files in `/etc/condor-ce/config.d` for two reasons:
1.  Historical: they were here before they were in `/usr/share/condor-ce` and we wanted some continuity.
2.  We want to communicate / highlight some config setting the admin may want to change.

Does either apply here?  The only potential tunable is `AUDIT_PAYLOAD_MAX_HOURS`, but I suspect that the admin would have to be such a deep expert to want to tune that they don't need the help text.

I'm tempted to just suggest removing this file completely; seems it would make things simpler.